src/_blog/rust-libp2p-browser-to-server.md

```diff
@@ -0,0 +1,61 @@
+---
+tags:
+  - rust-libp2p
+  - rust
+  - libp2p
+  - browser
+  - webrtc
+  - webassembly
+  - wasm
+title: Rust-libp2p in the Browser with WebRTC!
+description: News about how rust-libp2p now connects with the network via WebRTC
+date: 2023-11-07
+permalink: ''
+translationKey: ''
+header_image: /rust-libp2p.jpeg
+author: DougAnderson444
+---
+
+# Rust-libp2p WebRTC now available in the Browser
+
+WebAssembly WebRTC is now available for use as a browser transport with rust-libp2p! Before now, WebRTC was only available on the server in `rust-libp2p`, but after months of coding and reviews, we are pleased to annouce it is available on both the server and in the browser.
```

Comment:
```suggestion
rust-libp2p running in the browser can now establish WebRTC connections! Before now, WebRTC was only available on the server in `rust-libp2p`, but after months of coding and reviews, we are pleased to annouce it is available on both the server and in the browser.
```

Would be easier for me to understand. What do you think?

Reply:
I like it. It's a lot of terms to squeeze into one breath, I think your rendition does the trick very nicely